{"body": "I *think* this is the same bug that was fixed in [bug 1416003](https://bugzilla.mozilla.org/show_bug.cgi?id=1416003).", "body_text": "I think this is the same bug that was fixed in bug 1416003.", "url": "https://api.github.com/repos/mozilla/normandy/issues/comments/361719715", "created_at": "2018-01-30T20:12:57Z", "author_association": "MEMBER", "html_url": "https://github.com/mozilla/normandy/issues/798#issuecomment-361719715", "updated_at": "2018-01-30T20:12:57Z", "user": {"following_url": "https://api.github.com/users/mythmon/following{/other_user}", "events_url": "https://api.github.com/users/mythmon/events{/privacy}", "organizations_url": "https://api.github.com/users/mythmon/orgs", "url": "https://api.github.com/users/mythmon", "gists_url": "https://api.github.com/users/mythmon/gists{/gist_id}", "html_url": "https://github.com/mythmon", "subscriptions_url": "https://api.github.com/users/mythmon/subscriptions", "avatar_url": "https://avatars3.githubusercontent.com/u/305049?v=4", "repos_url": "https://api.github.com/users/mythmon/repos", "received_events_url": "https://api.github.com/users/mythmon/received_events", "gravatar_id": "", "starred_url": "https://api.github.com/users/mythmon/starred{/owner}{/repo}", "site_admin": false, "login": "mythmon", "type": "User", "id": 305049, "followers_url": "https://api.github.com/users/mythmon/followers"}, "body_html": "<p>I <em>think</em> this is the same bug that was fixed in <a href=\"https://bugzilla.mozilla.org/show_bug.cgi?id=1416003\" rel=\"nofollow\">bug 1416003</a>.</p>", "id": 361719715, "issue_url": "https://api.github.com/repos/mozilla/normandy/issues/798"}